packages/webapp/src/webview/state/reducers.ts

```diff
@@ -215,6 +220,8 @@ function setActiveTreeReducer(newState: AppState, action: SetActiveTree): AppSta
  * @returns new state with changes
  */
 function setQueryValueReducer(newState: AppState, action: SetQueryValue): AppState {
+    newState.updatedGuidedAnswerTrees = [];
+    newState.currentOffset = 0;
```

Comment:
If you follow my proposal to not introduce additional properties `updatedGuidedAnswerTrees` and `currentOffset`, on the state, you should re-initialize `newState.guidedAnswerTreeSearchResult` when we do a search.
Instead of doing it here, I'd do it in `searchTreeReducer()`

Reply:
Changed, thanks!